setup.py

```diff
@@ -121,7 +121,7 @@ def get_version():
         'requests>=2.20.0',
         'jinja2==2.11.3',
         'pyyaml==5.4.1',
-        'certifi==2017.4.17',
+        'certifi==2022.9.24',
```

Comment:
Looks like a big jump there..not sure the implications of this one.  Have you tested anything with this one?

Reply:
No, I am going to merge them all and then test it again before taking a stable tag.. that way I can revert things.